src/Contracts/SearchQuery.php

```diff
@@ -26,6 +26,7 @@ class SearchQuery
     private ?int $hitsPerPage;
     private ?int $page;
     private ?array $vector;
+    private ?array $attributesToSearchOn;
```

Comment:
I'd instead initialize props as null to avoid null coalescing in `toArray`

Reply:
You mean: `private ?array $attributesToSearchOn = null;`?

Reply:
Yup :)

Reply:
But then would better to do this everywhere if needed

Reply:
I made it for the `attributesToSearchOn` and I'll raise an issue to do it for the others :)

Reply:
Done: https://github.com/meilisearch/meilisearch-php/issues/547